ios/brave-ios/Sources/Brave/WebFilters/AdblockResourceDownloader.swift

```diff
@@ -31,7 +31,7 @@ public actor AdblockResourceDownloader: Sendable {
   }
 
   /// Start fetching resources
-  public func startFetching() {
+  public func startFetching() async {
```

Comment:
you don't seem to be adding any await but rather a Task. So maybe this `async` is not needed?